hawkbit-runtime/.sandbox/README.md

```diff
@@ -0,0 +1,61 @@
+hawkBit Sandbox
+===
+
+## Try out the update server in our hawkBit sandbox
+
+- try out Management UI https://hawkbit.eclipse.org/UI (username: demo, passwd: demo)
+- try out Management API https://hawkbit.eclipse.org/rest/v1/targets (don't forget basic auth header; username: demo, passwd: demo)
+- try out DDI API https://hawkbit.eclipse.org/DEFAULT/controller/v1/MYTESTDEVICE 
+
+
+## Disclaimer
```

Comment:
I think you don't need the disclaimer here since this is only the manual how to run the sandbox